src/build123d/topology.py

```diff
@@ -7303,7 +7317,7 @@ def _connect_to(self, other: Joint, **kwargs):  # pragma: no cover
         self.connected_to = other
 
     @abstractmethod
-    def connect_to(self, other: Joint, **kwargs):
+    def connect_to(self, other: Joint, *args, **kwargs):
```

Comment:
Same comment as above.  Should this be `def connect_to(self, other: Joint, *, **kwargs):`? Seems redundant.

Reply:
Resolved by dropping existing variadic per other comment thread instead of adding bare star.